ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -406,6 +408,21 @@ const triggerSet: TriggerSet<Data> = {
       },
       default: 'xsct',
     },
+    {
+      id: 'pangenesisFirstTower',
+      name: {
+        en: 'Pangenesis: First Towers',
+      },
+      type: 'select',
+      options: {
+        en: {
+          'Call Required Swaps Only': 'agnostic',
+          '0+2 (Conga Line/HRT)': 'not',
+          '1+2 (Yuki)': 'one',
```

Comment:
Is this also "Rinon" strat? Or am I misremembering?

Reply:
I just watched Rinon's guide, and yes it is.  I'll add that.